src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForOcrValidationFailedOnServerSide.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.tasks;
+
+import com.google.common.io.Resources;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.TestPropertySource;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.ProcessEvent;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.FormData;
+
+import java.nio.charset.Charset;
+import java.util.List;
+
+import static com.google.common.io.Resources.getResource;
+import static com.jayway.jsonpath.JsonPath.parse;
+import static java.util.stream.Collectors.toList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.given;
+import static uk.gov.hmcts.reform.bulkscanprocessor.entity.Status.CREATED;
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+import static uk.gov.hmcts.reform.bulkscanprocessor.model.common.Event.ZIPFILE_PROCESSING_STARTED;
+
+@IntegrationTest
+@TestPropertySource(properties = {
+    "scheduling.task.scan.enabled=true"
+})
+class BlobProcessorTaskTestForOcrValidationFailedOnServerSide extends ProcessorTestSuiteForOcrValidation {
```

Comment:
I am not sure if this test is required here. I mean the integration was already proved above, and this repeats something, which I guess is already tested on the unit test level? 

But as a newcomer not sure how you approach those integration tests, so it's fine if you cover the functionality more thoroughly, but I think it's against the testing pyramid (a bit). 

My another point it's difficult to understand why this test returns warning and above one doesn't. Why retrying for the second time is special? I found it is driven by the config, but maybe it could be added as the comment

Reply:
In the current production code in case of unavailability of the OCR validation endpoint on the service side the warning is raised straight away. 

In this PR (according the task) there are several attempts (the number of defined by the config) to hit this endpoint (delay between the attempts is also defined by the config). If the number of attempts is expired - the warning is raised and processing of the file goes on.